examples/llm/benchmarks/plot_pareto.py

```
@@ -0,0 +1,275 @@
+    if deployment_config.get("mode") == "aggregated":
+        return deployment_config.get("tensor_parallelism") * deployment_config.get(
+            "data_parallelism"
+        )
+    else:
+        return deployment_config.get(
+            "prefill_tensor_parallelism"
+        ) * deployment_config.get("prefill_data_parallelism") + deployment_config.get(
+            "decode_tensor_parallelism"
+        ) * deployment_config.get(
+            "decode_data_parallelism"
+        )
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)